services/efs-guardian/src/simcore_service_efs_guardian/services/efs_manager_utils.py

```diff
@@ -0,0 +1,47 @@
+import asyncio
+import logging
+
+from pydantic import ByteSize
+
+_logger = logging.getLogger(__name__)
+
+
+async def get_size_bash_async(path) -> ByteSize:
```

Comment:
Why not do it with pathlib? Inside file_utils.py there are some functions that show you how to scan all subduers, so something similar can be done. I would also place it there for everybody to use.
it would also be less error prone

Reply:
Guardian will do this call often, so using this executable should be more effective.